generalized_linear_regression/generalized_linear_regression.md

```diff
@@ -0,0 +1,528 @@
+negative values indicate that the outcome is unlikely, and positive values indicate that the outcome is likely. 
+
+$$
+logit(\text{sepsis}) = \beta_0 + \beta_{\text{temp}} * \text{temp} + e
+$$ 
+
+Now we can estimate a linear model on the transformed outcome.
+Then, after we get the estimates for the coefficients $\beta_0$ and $\beta_{\text{temp}}$, we can covert the equation back to the original scale to get predictions that will be probabilities (0-1).
+
+If we were to actually try to calculate this for our fake data by hand, though, we would quickly run into a problem. 
+The data are observed cases where the patient either did or did not have sepsis, so they're all 0 or 1 exactly in the data. 
+The log odds of 0 and 1 will be negative infinity and positive infinity, respectively. 
+We can't actually work with those numbers. 
+But the computer can!
+
+<div class = "behind-the-scenes">
+<b style="color: rgb(var(--color-highlight));">Behind the scenes</b><br>
+
+You certainly don't need to understand how your statistical software arrives at its solution in order to run and interpret generalized linear models.
+But if you're curious, one of the most common methods is called [maximum likelihood estimation](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation).
+
+</div>
+
+If we estimate the model with the logit link function, and then convert the predictions back to a 0-1 scale, we get something like this:
+
+![The same scatterplot showing temperature on the x-axis and sepsis (0 or 1) on the y-axis, but the line of best fit is now a sigmoidal curve, approaching but never reaching 0 for low values of temperature, and then curving up gently as temperature increases.](media/logit_prediction.png)
+
+``` -R code for the above plot
+ggplot(data, aes(y=sepsis, x=temp)) + 
+  geom_point() + 
+  theme_bw() + 
+  labs(y = "Sepsis", x = "Temperature") + 
+  scale_y_continuous(breaks = c(0,1))  + 
+  stat_smooth(method = "glm", method.args = list(family = "binomial"))
+```
+
+That looks much better!
+The predicted values now respect the boundaries on our outcome variable -- they can never go below 0 or above 1. 
+
+## Other Link Functions
+
+Logistic regression (regression using a logit link function) works well for binary outcomes.
+But what about other kinds of outcome variables?
+
+If your outcome is counts of something, you can use a **Poisson** or **Negative Binomial** link function. 
+For example, perhaps your outcome is the number of [Adverse Childhood Experiences (ACEs)](https://www.cdc.gov/violenceprevention/aces/fastfact.html) per patient. 
+It's not possible for someone to have fewer than 0 ACEs, so that variable is bound at 0. 
+It's also measured as a count (how many total ACEs experienced), so no one will have 1.2 ACEs, for example. 
+
+For outcomes bounded at 0 and 1 (like a probability or proportion), you can use logistic regression, but a **beta** link function may be more appropriate.
```

Comment:
I was initially confused by this -- since I didn't immediately connect that this doesn't include our binary variables, since those, too, are bounded at 0 and 1. Is it correct to say that here you're referring specifically to variables that are bounded but continuous within those bounds?

Reply:
I think I addressed this, lemme know if you still find it confusing.

Reply:
That one extra word makes all the difference! LGTM!